crates/forge/tests/cli/alphanet.rs

```diff
@@ -0,0 +1,19 @@
+// Ensure we can run basic counter tests with EOF support.
+#[cfg(not(windows))]
+forgetest_init!(test_eof_flag, |prj, cmd| {
```

Comment:
This test fails on macos CI https://github.com/foundry-rs/foundry/actions/runs/11158029813/job/31013451682